.github/workflows/release-production.yml

```diff
@@ -64,3 +64,28 @@ jobs:
           name: ${{ env.PACKAGE_NAME }}-v${{env.TAG}}
           path: ./*.tgz
           if-no-files-found: error
+
+  helm-chart-release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Install Helm
+        uses: azure/setup-helm@v3
+
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.5.0
+        with:
+          charts_dir: charts/
```

Comment:
With your rename of the main charts folder I don't think this would be needed

Reply:
@Nana-EC 
Yes, the renaming of the charts folder is not necessary. But need to keep chart manifests in one more deeper path, i.e. `helm-chart/xx/` directory to use chart releaser action recommended by helm https://helm.sh/docs/howto/chart_releaser_action/
https://github.com/helm/chart-releaser-action/blob/main/cr.sh#L268

If we cannot change the chart path depth, it means we cannot use this action and rather need to use low level script.

But i suggest to have one more depth for chart path and use this upstream action.
What is your opinion?

Reply:
@Nana-EC sorry, i was confused a bit.
Yes, definetly `charts/` is the default value so no need to specify explicitly
I will remove that